src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <label class="icon">
+        <a href="#home">
+          <img src="./images/logo.png"
+            alt="Moyo icon"
+          >
+        </a>  
+      </label>  
+      <nav>
+        <ul class="nav">
+          <li class="nav__item first">
```

Comment:
```suggestion
          <li class="nav__item">
```
Why do you need this class? use pseudo-class `:first-child` instead